cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerHttpHandlerInternal.java

```diff
@@ -45,6 +51,8 @@
 @Singleton
 @Path(Constants.Gateway.INTERNAL_API_VERSION_3 + "/worker")
 public class TaskWorkerHttpHandlerInternal extends AbstractLogHttpHandler {
+  static final String METADATA_SERVER_GET_TOKEN_URL =
+    "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token";
```

Comment:
Hard-coding a GKE specific URL into the OSS code isnt very friendly. Can we turn this into a property in cdap-site so OSS customers using other Kubernetes engines can still leverage this work?

Reply:
Thx for the suggestion. Now can be defined in cr config.